src/vm/method.hpp

```diff
@@ -258,6 +258,17 @@ class MethodDesc
         return (m_bFlags2 & enum_flag2_HasStableEntryPoint) != 0;
     }
 
+    inline BOOL HasStableEntryPoint1()
+    {
+        LIMITED_METHOD_DAC_CONTRACT;
+
+#if defined(FEATURE_JIT_PITCHING)
+        if (IsPitchable() && HasTemporaryEntryPoint())
+           return FALSE;
+#endif
+        return HasStableEntryPoint();
+    }
+
```

Comment:
@noahfalk Thanks a lot! Unfortunately I'll go to vacation for next two weeks. By the way, my measurements show that about 70% of code are pitched now on the CoreCLR test suite.